### ISSUE TITLE
Word embeddings for the LSTM?

### ISSUE THREAD

**Issue by littlewine** [HazyResearch/snorkel#963]
Is there some way I can use pretrained word embeddings in the end discriminative model (reRNN) ?

I have been looking through the code and docs but it seems unclear to me whether there's a way to do so, without modifying/extending the models within snorkel/learning

**Comment by ajratner** [comment]
@henryre ?

**Comment by littlewine** [comment]
Is there maybe an easier solution for the PyTorch models?
I can see that both models are using a [SymbolTable](https://github.com/HazyResearch/snorkel/blob/master/snorkel/learning/pytorch/rnn/rnn_base.py#L88) as a word_dict, but I am not sure how it should be modified for accepting the pre-trained word embeddings.

**Comment by ajratner** [comment]
@jason-fries @stephenbach @jason-fries any quick pointers / thoughts here?  If not, PRs always welcome :)!

**Comment by ajratner** [comment]
Closing this for now- happy to accept PRs, and/or reopen this!

**Comment by rjurney** [comment]
It seems that you could easily use a preprocessor to calculate / encode a document embedding and then plot the distribution of labels of each dimension of that embedding to create LFs.
